src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -43,14 +43,15 @@ const MapView: React.FC = () => {
     setShowModal(false);
     switch (reservationType) {
       case ReservationModalType.OPEN:
-        //set block status to reserved
-        protectedApiClient.makeReservation(activeBlockId);
+        //reserve block
+        protectedApiClient.makeReservation(activeBlockId); 
         break;
       case ReservationModalType.RESERVED:
-        //set block status to open
+        //release block
+        protectedApiClient.releaseReservation(activeBlockId);
         break;
       default:
-        //block clicked not owned/open
+        //block clicked not owned/open so do nothing
```

Comment:
Will this condition ever be hit? if so we should try to disable any buttons or interactions that would allow this.